packages/api/src/hooks/useBalance.tsx

```diff
@@ -0,0 +1,28 @@
+import { useFetch } from '@deriv/api';
+import { useMemo } from 'react';
+
+/** A custom hook that gets the balance for user accounts.
+ * If no account argument is provided, by default the hook will return the balance of all accounts.
+ *
+ * @param account {string} - If set to 'all', return the balances of all accounts one by one.
+ *                           If set to 'current', return the balance of current account.
+ *                           If set as an account id, return the balance of that account.
+ *                           Default is set to 'all'.
+ */
+const useBalance = (account?: string) => {
+    const { data: balance_data, ...rest } = useFetch('balance', {
+        payload: { account: account ?? 'all' },
```

Comment:
@adrienne-deriv Maybe use `option` to add a fetch interval until we have a subscription? 🤔

Reply:
Dont we have subscription already?

![image](https://github.com/binary-com/deriv-app/assets/125247833/c8afdb34-69c0-44ef-9563-e8485e772f96)

Reply:
Clarified with @yashim-deriv, when `all` is passed as payload to `account`, it is not subscrible-able, but when a loginid is passed instead, the endpoint is subscribe-able

Reply:
We can only subscribe once to each endpoint, Otherwise, we will get an error from BE that you are already subscribed. Since the client store is in the background subscribing to balance we can't subscribe again here. Instead, we will do a periodic fetch for now until we can serve the wallet without the core package.